### PR TITLE
Improvements around playlist

### PR DESCRIPTION
**Pull request by wutschel**
## Description

Choose correct default thumb for the Music and Video playlist.

## Summary for release notes

Bugfix: Show correct default thumb in movie/music playlists

### CODE REVIEW

**Comment by wutschel** [comment]
Should be merged after https://github.com/xbmc/Official-Kodi-Remote-iOS/pull/539 and needs a minor update to handle a third `playerID` which is introduced it.

**Comment by kambala-decapitator** [comment]
FYI #539 has already been merged

**Comment by wutschel** [comment]
Updated and rebased to master. Good for review.

File: XBMC Remote/NowPlaying.m [hunk 1]
```diff
@@ -2097,22 +2097,26 @@ - (UITableViewCell*)tableView:(UITableView*)tableView cellForRowAtIndexPath:(NSI
     else if ([item[@"type"] isEqualToString:@"movie"]) {
         subLabel.text = [NSString stringWithFormat:@"%@", item[@"genre"]];
     }
+    UIImage *defaultThumb = [UIImage imageNamed:@"nocover_filemode"];
```
Comment (kambala-decapitator) [inline XBMC Remote/NowPlaying.m:2087]: makes more sense to assign this in `default`

**Review by kambala-decapitator — CHANGES_REQUESTED** [review]

File: XBMC Remote/NowPlaying.m (hunk 1 above)
Comment (wutschel) [reply XBMC Remote/NowPlaying.m]: Agreed, also will set a default for `cornerLabel.text`.

**Review by kambala-decapitator — APPROVED** [review]

**Comment by wutschel** [comment]
Squashed and ready to go.

**Pull request merged**